src/server_manager/web_app/app.ts

```diff
@@ -137,10 +124,37 @@ type DigitalOceanSessionFactory = (accessToken: string) => digitalocean_api.Digi
 type DigitalOceanServerRepositoryFactory = (session: digitalocean_api.DigitalOceanSession) =>
     server.ManagedServerRepository;
 
+export class NotificationManager {
```

Comment:
Can you make this its own component outside approot with the show error and show notifications methods?